actions.js

```diff
@@ -362,8 +362,8 @@ module.exports = class Action {
 
     getLatestBlock(){
         return this._blockchainHandler().getLatestBlock()
-        .then(block => {
-            logger.console("Latest block :\n" + JSON.stringify(block,null,4));
+        .then(data => {
+            logger.console("Latest block :\n" + JSON.stringify(data.body.result.Block,null,4));
```

Comment:
It's better we encapsulate  these kind of objects and data (data.body.result.Block)  inside the corresponding method of the Account or any other lower layer classes and just return the specific wanted data.

Reply:
noted. I will revert back with the changes